src/main/java/com/google/step/servlets/JsonFormatter.java

```diff
@@ -200,17 +210,10 @@ public static String getUserJson(
     return userMaps;
   }
 
-  // TODO
```

Comment:
keep the TODO?

Reply:
I merged master into this branch since this function is implemented in master already.